### PR TITLE
Start on splitting web / desktop version

### PR DESCRIPTION
**Pull request by shama**
This is a start on a web version. It makes `app.js` more platform agnostic and creates a `desktop.js` and `web.js` for the entry points to each version.

Although further work needs to be done to get it running on the web. I'm hitting this error:
```
Error: Cannot find module '/Users/Kyle/Documents/www/friends/node_modules/protocol-buffers/brfs' from '/Users/Kyle/Documents/www/friends'
```
and I imagine a few others are not web friendly. Still looking into solutions around those.

---

`npm run web` will start a server on `localhost:9966`.

### CODE REVIEW

**Comment by ngoldman** [comment]
@shama cool! if you run `standard -F` locally it should fix most of the linting errors.

**Comment by shama** [comment]
@ngoldman Oh nice, I didnt know about `standard -F`. Fixed those errors, thanks!

**Comment by ngoldman** [comment]
@shama I think it just wraps https://github.com/maxogden/standard-format

File: package.json [hunk 1]
```diff
@@ -74,6 +75,9 @@
     "type": "git",
     "url": "https://github.com/moose-team/friends.git"
   },
+  "browserif": {
```
Comment (ngoldman) [inline package.json]: missing a `y` I think

File: web.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module.exports = Web
+
+var inherits = require('util').inherits
+var App = require('./app.js')
+
+inherits(Web, App)
```
Comment (ngoldman) [inline web.js]: :heart:

File: package.json (hunk 1 above)
Comment (shama) [inline package.json]: oops thanks!

**Comment by ngoldman** [comment]
Looks good to me, here we go!

**Comment by Flet** [comment]
@shama awesome start!

**Pull request merged**